packer/install-dependencies.sh

```diff
@@ -39,7 +40,10 @@ for CNI_PLUGIN_VERSION in $CNI_PLUGIN_VERSIONS; do
     downloadCNI
 done
 
-installContainerd
+for CONTAINERD_VERSION in $CONTAINERD_VERSIONS; do
```

Comment:
Instead of actually doing the install, we're just downloading the artifacts in the VHD.